From patchwork Tue May 12 10:32:33 2020
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 8bit
X-Patchwork-Submitter: =?utf-8?q?Philippe_Mathieu-Daud=C3=A9?=
 <philmd@redhat.com>
X-Patchwork-Id: 11542877
Return-Path: <SRS0=B8Dm=62=vger.kernel.org=kvm-owner@kernel.org>
Received: from mail.kernel.org (pdx-korg-mail-1.web.codeaurora.org
 [172.30.200.123])
	by pdx-korg-patchwork-2.web.codeaurora.org (Postfix) with ESMTP id A38C381
	for <patchwork-kvm@patchwork.kernel.org>;
 Tue, 12 May 2020 10:32:52 +0000 (UTC)
Received: from vger.kernel.org (vger.kernel.org [23.128.96.18])
	by mail.kernel.org (Postfix) with ESMTP id 8BEB520731
	for <patchwork-kvm@patchwork.kernel.org>;
 Tue, 12 May 2020 10:32:52 +0000 (UTC)
Authentication-Results: mail.kernel.org;
	dkim=pass (1024-bit key) header.d=redhat.com header.i=@redhat.com
 header.b="YTX1vGjc"
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand
        id S1729529AbgELKcv (ORCPT
        <rfc822;patchwork-kvm@patchwork.kernel.org>);
        Tue, 12 May 2020 06:32:51 -0400
Received: from us-smtp-delivery-1.mimecast.com ([207.211.31.120]:34903 "EHLO
        us-smtp-1.mimecast.com" rhost-flags-OK-OK-OK-FAIL) by vger.kernel.org
        with ESMTP id S1729429AbgELKcu (ORCPT <rfc822;kvm@vger.kernel.org>);
        Tue, 12 May 2020 06:32:50 -0400
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=redhat.com;
        s=mimecast20190719; t=1589279569;
        h=from:from:reply-to:subject:subject:date:date:message-id:message-id:
         to:to:cc:cc:mime-version:mime-version:
  content-type:content-type:content-type:
         content-transfer-encoding:content-transfer-encoding:
         in-reply-to:in-reply-to:references:references;
        bh=o+SFxNiQTSKSfzcy81S2xitOyf6spOQEd9fCWLhnCeY=;
        b=YTX1vGjcF6s24R7sSz0yGitSRFfWV6lJFWD7VA0ILNRdtZJVYnpcYAb63Bwpqe0ooJpjwJ
        9yFuWJ9lS4ywgqj+sV1msAbaTf/DsBqeI7M2evwYYWuWTQYrgNH7K5G/DTUoTjp0DYKVXj
        LTvCIQUWjsn/Icl4+tdjFI/4pAQbvuA=
Received: from mail-wm1-f71.google.com (mail-wm1-f71.google.com
 [209.85.128.71]) (Using TLS) by relay.mimecast.com with ESMTP id
 us-mta-239-MgccDOUEOceGvkulRkuBgg-1; Tue, 12 May 2020 06:32:48 -0400
X-MC-Unique: MgccDOUEOceGvkulRkuBgg-1
Received: by mail-wm1-f71.google.com with SMTP id e15so935652wme.1
        for <kvm@vger.kernel.org>; Tue, 12 May 2020 03:32:47 -0700 (PDT)
X-Google-DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
        d=1e100.net; s=20161025;
        h=x-gm-message-state:from:to:cc:subject:date:message-id:in-reply-to
         :references:mime-version:content-transfer-encoding;
        bh=o+SFxNiQTSKSfzcy81S2xitOyf6spOQEd9fCWLhnCeY=;
        b=gm/gYdwMAhiOyAP8GaPYC80pouJUjd51ZlV7iF9ot7Yvz9+hDQ7+4LaSUrG5yu3Qt/
         hrR3efkI9WjoPAI2XQc9bDeqIyfR1w7BZkrKTk2qAKzAoOvMKaCxE5c0XKfwISi/KdfH
         QoGA4TddStjCsNuVsEUBl2hQi/tpzNw2nRmicqTMb2DSvmnesQ5Qw0Ic5O98PEM6Dov/
         HVO1veenmRc9pRdEaMYP9KxbZRwYuV/WTno80mSeDWLKo5QrhPtBfXweNQD8e/CGUy8U
         0LWZVt8b9g/uhGquA6mxMrAUWAKNXFX64c9/cDsgrV9TLV+MSgt8Eacz2l3ctewIqOV5
         yHiA==
X-Gm-Message-State: AGi0PuYVdB/vbf+b0WTzPsCBkh+aNwNoIDNnGC8U0cQvwCCDmPcO9iJL
        dJND69jLCUCvoJrE3s3AuE7lpnMInhjWsvXSP9qUn1OXZJO3h3l/SGML37mpk4uXP+JGbTlNXoi
        PdAS0DLoQQ0tJ
X-Received: by 2002:adf:cc81:: with SMTP id p1mr23275929wrj.192.1589279565427;
        Tue, 12 May 2020 03:32:45 -0700 (PDT)
X-Google-Smtp-Source: 
 APiQypIdq6xklNhE6zF2ex6riiZ2WEeaRBZwdbLSVo8fxAyW06Ghhyjcwp0kiKh5i+SK1pgUYQQIhQ==
X-Received: by 2002:adf:cc81:: with SMTP id p1mr23275905wrj.192.1589279565172;
        Tue, 12 May 2020 03:32:45 -0700 (PDT)
Received: from x1w.redhat.com (17.red-88-21-202.staticip.rima-tde.net.
 [88.21.202.17])
        by smtp.gmail.com with ESMTPSA id
 u16sm21863464wrq.17.2020.05.12.03.32.43
        (version=TLS1_3 cipher=TLS_AES_256_GCM_SHA384 bits=256/256);
        Tue, 12 May 2020 03:32:44 -0700 (PDT)
From: =?utf-8?q?Philippe_Mathieu-Daud=C3=A9?= <philmd@redhat.com>
To: qemu-devel@nongnu.org
Cc: Markus Armbruster <armbru@redhat.com>, John Snow <jsnow@redhat.com>,
 Paolo Bonzini <pbonzini@redhat.com>, qemu-block@nongnu.org,
 qemu-trivial@nongnu.org,
 =?utf-8?q?Philippe_Mathieu-Daud=C3=A9?= <philmd@redhat.com>,
 Cleber Rosa <crosa@redhat.com>, kvm@vger.kernel.org,
 Eduardo Habkost <ehabkost@redhat.com>,
 =?utf-8?q?Alex_Benn=C3=A9e?= <alex.bennee@linaro.org>,
 Marcelo Tosatti <mtosatti@redhat.com>, Stefan Hajnoczi <stefanha@redhat.com>,
 Fam Zheng <fam@euphon.net>,
 =?utf-8?q?Philippe_Mathieu-Daud=C3=A9?= <f4bug@amsat.org>
Subject: [PATCH v4 1/6] scripts/qemugdb: Remove shebang header
Date: Tue, 12 May 2020 12:32:33 +0200
Message-Id: <20200512103238.7078-2-philmd@redhat.com>
X-Mailer: git-send-email 2.21.3
In-Reply-To: <20200512103238.7078-1-philmd@redhat.com>
References: <20200512103238.7078-1-philmd@redhat.com>
MIME-Version: 1.0
Sender: kvm-owner@vger.kernel.org
Precedence: bulk
List-ID: <kvm.vger.kernel.org>
X-Mailing-List: kvm@vger.kernel.org

From: Philippe Mathieu-Daudé <f4bug@amsat.org>

These scripts are loaded as plugin by GDB (and they don't
have any __main__ entry point). Remove the shebang header.

Acked-by: Alex Bennée <alex.bennee@linaro.org>
Signed-off-by: Philippe Mathieu-Daudé <f4bug@amsat.org>
---
 scripts/qemugdb/__init__.py  | 3 +--
 scripts/qemugdb/aio.py       | 3 +--
 scripts/qemugdb/coroutine.py | 3 +--
 scripts/qemugdb/mtree.py     | 4 +---
 scripts/qemugdb/tcg.py       | 1 -
 scripts/qemugdb/timers.py    | 1 -
 6 files changed, 4 insertions(+), 11 deletions(-)

diff --git a/scripts/qemugdb/__init__.py b/scripts/qemugdb/__init__.py
index 969f552b26..da8ff612e5 100644
--- a/scripts/qemugdb/__init__.py
+++ b/scripts/qemugdb/__init__.py
@@ -1,5 +1,4 @@
-#!/usr/bin/python
-
+#
 # GDB debugging support
 #
 # Copyright (c) 2015 Linaro Ltd
diff --git a/scripts/qemugdb/aio.py b/scripts/qemugdb/aio.py
index 2ba00c4444..d7c1ba0c28 100644
--- a/scripts/qemugdb/aio.py
+++ b/scripts/qemugdb/aio.py
@@ -1,5 +1,4 @@
-#!/usr/bin/python
-
+#
 # GDB debugging support: aio/iohandler debug
 #
 # Copyright (c) 2015 Red Hat, Inc.
diff --git a/scripts/qemugdb/coroutine.py b/scripts/qemugdb/coroutine.py
index 41e079d0e2..db61389022 100644
--- a/scripts/qemugdb/coroutine.py
+++ b/scripts/qemugdb/coroutine.py
@@ -1,5 +1,4 @@
-#!/usr/bin/python
-
+#
 # GDB debugging support
 #
 # Copyright 2012 Red Hat, Inc. and/or its affiliates
diff --git a/scripts/qemugdb/mtree.py b/scripts/qemugdb/mtree.py
index 3030a60d3f..8fe42c3c12 100644
--- a/scripts/qemugdb/mtree.py
+++ b/scripts/qemugdb/mtree.py
@@ -1,5 +1,4 @@
-#!/usr/bin/python
-
+#
 # GDB debugging support
 #
 # Copyright 2012 Red Hat, Inc. and/or its affiliates
@@ -84,4 +83,3 @@ def print_item(self, ptr, offset = gdb.Value(0), level = 0):
         while not isnull(subregion):
             self.print_item(subregion, addr, level)
             subregion = subregion['subregions_link']['tqe_next']
-
diff --git a/scripts/qemugdb/tcg.py b/scripts/qemugdb/tcg.py
index 18880fc9a7..16c03c06a9 100644
--- a/scripts/qemugdb/tcg.py
+++ b/scripts/qemugdb/tcg.py
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # -*- coding: utf-8 -*-
 #
 # GDB debugging support, TCG status
diff --git a/scripts/qemugdb/timers.py b/scripts/qemugdb/timers.py
index f0e132d27a..46537b27cf 100644
--- a/scripts/qemugdb/timers.py
+++ b/scripts/qemugdb/timers.py
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # -*- coding: utf-8 -*-
 # GDB debugging support
 #

From patchwork Tue May 12 10:32:34 2020
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 8bit
X-Patchwork-Submitter: =?utf-8?q?Philippe_Mathieu-Daud=C3=A9?=
 <philmd@redhat.com>
X-Patchwork-Id: 11542879
Return-Path: <SRS0=B8Dm=62=vger.kernel.org=kvm-owner@kernel.org>
Received: from mail.kernel.org (pdx-korg-mail-1.web.codeaurora.org
 [172.30.200.123])
	by pdx-korg-patchwork-2.web.codeaurora.org (Postfix) with ESMTP id 9A530139A
	for <patchwork-kvm@patchwork.kernel.org>;
 Tue, 12 May 2020 10:32:57 +0000 (UTC)
Received: from vger.kernel.org (vger.kernel.org [23.128.96.18])
	by mail.kernel.org (Postfix) with ESMTP id 840B520731
	for <patchwork-kvm@patchwork.kernel.org>;
 Tue, 12 May 2020 10:32:57 +0000 (UTC)
Authentication-Results: mail.kernel.org;
	dkim=pass (1024-bit key) header.d=redhat.com header.i=@redhat.com
 header.b="hii1X1x+"
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand
        id S1729527AbgELKc4 (ORCPT
        <rfc822;patchwork-kvm@patchwork.kernel.org>);
        Tue, 12 May 2020 06:32:56 -0400
Received: from us-smtp-1.mimecast.com ([205.139.110.61]:46967 "EHLO
        us-smtp-1.mimecast.com" rhost-flags-OK-OK-OK-OK) by vger.kernel.org
        with ESMTP id S1729365AbgELKcz (ORCPT <rfc822;kvm@vger.kernel.org>);
        Tue, 12 May 2020 06:32:55 -0400
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=redhat.com;
        s=mimecast20190719; t=1589279575;
        h=from:from:reply-to:subject:subject:date:date:message-id:message-id:
         to:to:cc:cc:mime-version:mime-version:
  content-type:content-type:content-type:
         content-transfer-encoding:content-transfer-encoding:
         in-reply-to:in-reply-to:references:references;
        bh=zOtCARhB88XtH+7DaaroGpTjVIhIb43b3LSLjds5GUw=;
        b=hii1X1x+Cawdu+B+Y7OKr81V08+1f4gIAKCAt9d5OSCb2GtLNqLPR/tgMpDaBQSZqDpjdR
        en5XVJGiBDeELQnCNCdZp210Z7EkYn04BVZV0jAY35DgTvJOkD6q6A2eJy9G+Ub74WNJIC
        UDr69wvtIBRQw5k7JTKJC0Bi2eHRpOA=
Received: from mail-wr1-f72.google.com (mail-wr1-f72.google.com
 [209.85.221.72]) (Using TLS) by relay.mimecast.com with ESMTP id
 us-mta-22-FLSWtuFRMQ-KrGOtVS5XTg-1; Tue, 12 May 2020 06:32:51 -0400
X-MC-Unique: FLSWtuFRMQ-KrGOtVS5XTg-1
Received: by mail-wr1-f72.google.com with SMTP id 37so3496548wrc.4
        for <kvm@vger.kernel.org>; Tue, 12 May 2020 03:32:51 -0700 (PDT)
X-Google-DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
        d=1e100.net; s=20161025;
        h=x-gm-message-state:from:to:cc:subject:date:message-id:in-reply-to
         :references:mime-version:content-transfer-encoding;
        bh=zOtCARhB88XtH+7DaaroGpTjVIhIb43b3LSLjds5GUw=;
        b=KcVnv6JMHIFCyJNPk4jwrdZx0EenVaU9Rgf4+SWyuG25QxymFuS2WtCykcCde0oLt5
         wbGMoSDOf5Pe0su9bnTlu4jck0uosG/0HKbA6vLUUh6h9KF2t/J2ElBr7gs1HW2/t0o3
         rrfRnQMtxnTmwfO2IANEeyidWTwH7blWnfGKXmdm8O6M8YdV1k07xjS3a5h7KqNFgLd0
         dqavCz08LYYTOHk0rRI9eaFpjG/hxgtJxoBAPB5G1A+BqGFjAH7c9bCbGMpfHw5lPupR
         3PEsn5Kgh1vz/rFhTZUJwvvntfHQspco8s9NTUKJ/Co+hnRo+T2+TcRJu0+Of71H6FbV
         7XiQ==
X-Gm-Message-State: AGi0PuaVhUzXV5y9hSG2ZWo/zNeaJYW5K9OUYYc9NpYAgwR+m+TfKYCb
        D4feNdBkoHz2DlNN8qZ5Elm7jKv7Hg0f/XBjphcR1RCMMktBHEBOYuGi3dBegB+lXIkGelvaDCD
        m/+u27e/foppR
X-Received: by 2002:adf:f3cc:: with SMTP id
 g12mr12858081wrp.427.1589279570203;
        Tue, 12 May 2020 03:32:50 -0700 (PDT)
X-Google-Smtp-Source: 
 APiQypIp4cKlTsev0nh13QNGo5ccEqxoYf5orH801v8Ck5Kq9gfAUjGI6X3+2CENt1I9LXed1D7kpQ==
X-Received: by 2002:adf:f3cc:: with SMTP id
 g12mr12858058wrp.427.1589279570025;
        Tue, 12 May 2020 03:32:50 -0700 (PDT)
Received: from x1w.redhat.com (17.red-88-21-202.staticip.rima-tde.net.
 [88.21.202.17])
        by smtp.gmail.com with ESMTPSA id
 u74sm31235914wmu.13.2020.05.12.03.32.48
        (version=TLS1_3 cipher=TLS_AES_256_GCM_SHA384 bits=256/256);
        Tue, 12 May 2020 03:32:49 -0700 (PDT)
From: =?utf-8?q?Philippe_Mathieu-Daud=C3=A9?= <philmd@redhat.com>
To: qemu-devel@nongnu.org
Cc: Markus Armbruster <armbru@redhat.com>, John Snow <jsnow@redhat.com>,
 Paolo Bonzini <pbonzini@redhat.com>, qemu-block@nongnu.org,
 qemu-trivial@nongnu.org,
 =?utf-8?q?Philippe_Mathieu-Daud=C3=A9?= <philmd@redhat.com>,
 Cleber Rosa <crosa@redhat.com>, kvm@vger.kernel.org,
 Eduardo Habkost <ehabkost@redhat.com>,
 =?utf-8?q?Alex_Benn=C3=A9e?= <alex.bennee@linaro.org>,
 Marcelo Tosatti <mtosatti@redhat.com>, Stefan Hajnoczi <stefanha@redhat.com>,
 Fam Zheng <fam@euphon.net>,
 =?utf-8?q?Philippe_Mathieu-Daud=C3=A9?= <f4bug@amsat.org>
Subject: [PATCH v4 2/6] scripts/qemu-gdb: Use Python 3 interpreter
Date: Tue, 12 May 2020 12:32:34 +0200
Message-Id: <20200512103238.7078-3-philmd@redhat.com>
X-Mailer: git-send-email 2.21.3
In-Reply-To: <20200512103238.7078-1-philmd@redhat.com>
References: <20200512103238.7078-1-philmd@redhat.com>
MIME-Version: 1.0
Sender: kvm-owner@vger.kernel.org
Precedence: bulk
List-ID: <kvm.vger.kernel.org>
X-Mailing-List: kvm@vger.kernel.org

From: Philippe Mathieu-Daudé <f4bug@amsat.org>

Signed-off-by: Philippe Mathieu-Daudé <f4bug@amsat.org>
---
 scripts/qemu-gdb.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/scripts/qemu-gdb.py b/scripts/qemu-gdb.py
index f2a305c42e..e0bfa7b5a4 100644
--- a/scripts/qemu-gdb.py
+++ b/scripts/qemu-gdb.py
@@ -1,5 +1,5 @@
-#!/usr/bin/python
-
+#!/usr/bin/env python3
+#
 # GDB debugging support
 #
 # Copyright 2012 Red Hat, Inc. and/or its affiliates

From patchwork Tue May 12 10:32:35 2020
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 8bit
X-Patchwork-Submitter: =?utf-8?q?Philippe_Mathieu-Daud=C3=A9?=
 <philmd@redhat.com>
X-Patchwork-Id: 11542881
Return-Path: <SRS0=B8Dm=62=vger.kernel.org=kvm-owner@kernel.org>
Received: from mail.kernel.org (pdx-korg-mail-1.web.codeaurora.org
 [172.30.200.123])
	by pdx-korg-patchwork-2.web.codeaurora.org (Postfix) with ESMTP id DF6CA913
	for <patchwork-kvm@patchwork.kernel.org>;
 Tue, 12 May 2020 10:33:04 +0000 (UTC)
Received: from vger.kernel.org (vger.kernel.org [23.128.96.18])
	by mail.kernel.org (Postfix) with ESMTP id C4AB820722
	for <patchwork-kvm@patchwork.kernel.org>;
 Tue, 12 May 2020 10:33:04 +0000 (UTC)
Authentication-Results: mail.kernel.org;
	dkim=pass (1024-bit key) header.d=redhat.com header.i=@redhat.com
 header.b="beGElXic"
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand
        id S1729537AbgELKdD (ORCPT
        <rfc822;patchwork-kvm@patchwork.kernel.org>);
        Tue, 12 May 2020 06:33:03 -0400
Received: from us-smtp-delivery-1.mimecast.com ([207.211.31.120]:55187 "EHLO
        us-smtp-1.mimecast.com" rhost-flags-OK-OK-OK-FAIL) by vger.kernel.org
        with ESMTP id S1726193AbgELKdC (ORCPT <rfc822;kvm@vger.kernel.org>);
        Tue, 12 May 2020 06:33:02 -0400
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=redhat.com;
        s=mimecast20190719; t=1589279581;
        h=from:from:reply-to:subject:subject:date:date:message-id:message-id:
         to:to:cc:cc:mime-version:mime-version:
  content-type:content-type:content-type:
         content-transfer-encoding:content-transfer-encoding:
         in-reply-to:in-reply-to:references:references;
        bh=oWSfUmBy5G7w8s0mFzIrpHSBiQEAL75l96FocmDeH+w=;
        b=beGElXichNoGlVPHOfVGmAr499YoorKbPvV4JXgufmwsi5Sl/bCoj4+cfiK++3EjwXWQth
        wPtyMCf1myp/HRe3ZIdBtCJnvJf4/xnWh9GIqyYoKwWWs+7BqJI44REayIZBfZIQi6masY
        znujiaP1TlEj2qdv4g5WDECotYw3CBo=
Received: from mail-wm1-f71.google.com (mail-wm1-f71.google.com
 [209.85.128.71]) (Using TLS) by relay.mimecast.com with ESMTP id
 us-mta-217-4PVQ_z-YNyGnm2Np0QjCDw-1; Tue, 12 May 2020 06:32:55 -0400
X-MC-Unique: 4PVQ_z-YNyGnm2Np0QjCDw-1
Received: by mail-wm1-f71.google.com with SMTP id p185so2451489wmp.5
        for <kvm@vger.kernel.org>; Tue, 12 May 2020 03:32:55 -0700 (PDT)
X-Google-DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
        d=1e100.net; s=20161025;
        h=x-gm-message-state:from:to:cc:subject:date:message-id:in-reply-to
         :references:mime-version:content-transfer-encoding;
        bh=oWSfUmBy5G7w8s0mFzIrpHSBiQEAL75l96FocmDeH+w=;
        b=VTo6whdNsOEwAb/HspuGiUGHJTTDjlkeVvFI1dTn9cViybeT2Gd7u+t89BOPed7VeN
         c4W8tF+Hn8HwHQyYR6TlLlphmrhlyEQdSlucpqiGHw+ukGv92V/BKxxhf7AN8CLhSCEw
         RRdcy9pEu7texlZyDClskhR65HpLb8vGgkPxL9yRu6Y+e+Bym8st6mxieQEy0GBh+zDr
         UHuHRfESSjcxBUgrGWkZbZ9CqAaK39VeR0GdmG+Md2oyRjLIdDfHhVvPDQWZMQi/gm88
         4KqZr7BxekKKik5mLZhWejer56jEZ1dd3WDLV+51ckNa8K42fzsEpDsky5TKy/XZFHHV
         0xXA==
X-Gm-Message-State: AGi0PubIzgF5TAeqJrTeapM9sPySEkpfccsE50rnYT3Cm196lzvKGHEy
        u79j6UfyvKH64/GT9cozn7InFJg0yTXc6QpB96aa9FsSquC7sFv0IVNZFbrYo58pFcWpHqWvtYe
        iIZX8LRb5oY8/
X-Received: by 2002:a5d:560c:: with SMTP id
 l12mr23822070wrv.309.1589279574898;
        Tue, 12 May 2020 03:32:54 -0700 (PDT)
X-Google-Smtp-Source: 
 APiQypIQhhMzrfHJzuqqGpCMtne83+qTrDIGPL9GtO3XV5eyvzYb43eEF0XvrPWMJEaysSiQNLomJg==
X-Received: by 2002:a5d:560c:: with SMTP id
 l12mr23822059wrv.309.1589279574750;
        Tue, 12 May 2020 03:32:54 -0700 (PDT)
Received: from x1w.redhat.com (17.red-88-21-202.staticip.rima-tde.net.
 [88.21.202.17])
        by smtp.gmail.com with ESMTPSA id
 b18sm1112431wrn.82.2020.05.12.03.32.53
        (version=TLS1_3 cipher=TLS_AES_256_GCM_SHA384 bits=256/256);
        Tue, 12 May 2020 03:32:54 -0700 (PDT)
From: =?utf-8?q?Philippe_Mathieu-Daud=C3=A9?= <philmd@redhat.com>
To: qemu-devel@nongnu.org
Cc: Markus Armbruster <armbru@redhat.com>, John Snow <jsnow@redhat.com>,
 Paolo Bonzini <pbonzini@redhat.com>, qemu-block@nongnu.org,
 qemu-trivial@nongnu.org,
 =?utf-8?q?Philippe_Mathieu-Daud=C3=A9?= <philmd@redhat.com>,
 Cleber Rosa <crosa@redhat.com>, kvm@vger.kernel.org,
 Eduardo Habkost <ehabkost@redhat.com>,
 =?utf-8?q?Alex_Benn=C3=A9e?= <alex.bennee@linaro.org>,
 Marcelo Tosatti <mtosatti@redhat.com>, Stefan Hajnoczi <stefanha@redhat.com>,
 Fam Zheng <fam@euphon.net>,
 =?utf-8?q?Philippe_Mathieu-Daud=C3=A9?= <f4bug@amsat.org>
Subject: [PATCH v4 3/6] scripts/qmp: Use Python 3 interpreter
Date: Tue, 12 May 2020 12:32:35 +0200
Message-Id: <20200512103238.7078-4-philmd@redhat.com>
X-Mailer: git-send-email 2.21.3
In-Reply-To: <20200512103238.7078-1-philmd@redhat.com>
References: <20200512103238.7078-1-philmd@redhat.com>
MIME-Version: 1.0
Sender: kvm-owner@vger.kernel.org
Precedence: bulk
List-ID: <kvm.vger.kernel.org>
X-Mailing-List: kvm@vger.kernel.org

From: Philippe Mathieu-Daudé <f4bug@amsat.org>

Signed-off-by: Philippe Mathieu-Daudé <f4bug@amsat.org>
---
 scripts/qmp/qom-get  | 2 +-
 scripts/qmp/qom-list | 2 +-
 scripts/qmp/qom-set  | 2 +-
 scripts/qmp/qom-tree | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/scripts/qmp/qom-get b/scripts/qmp/qom-get
index 007b4cd442..7c5ede91bb 100755
--- a/scripts/qmp/qom-get
+++ b/scripts/qmp/qom-get
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python3
 ##
 # QEMU Object Model test tools
 #
diff --git a/scripts/qmp/qom-list b/scripts/qmp/qom-list
index 03bda3446b..bb68fd65d4 100755
--- a/scripts/qmp/qom-list
+++ b/scripts/qmp/qom-list
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python3
 ##
 # QEMU Object Model test tools
 #
diff --git a/scripts/qmp/qom-set b/scripts/qmp/qom-set
index c37fe78b00..19881d85e9 100755
--- a/scripts/qmp/qom-set
+++ b/scripts/qmp/qom-set
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python3
 ##
 # QEMU Object Model test tools
 #
diff --git a/scripts/qmp/qom-tree b/scripts/qmp/qom-tree
index 1c8acf61e7..fa91147a03 100755
--- a/scripts/qmp/qom-tree
+++ b/scripts/qmp/qom-tree
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python3
 ##
 # QEMU Object Model test tools
 #

From patchwork Tue May 12 10:32:36 2020
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 8bit
X-Patchwork-Submitter: =?utf-8?q?Philippe_Mathieu-Daud=C3=A9?=
 <philmd@redhat.com>
X-Patchwork-Id: 11542883
Return-Path: <SRS0=B8Dm=62=vger.kernel.org=kvm-owner@kernel.org>
Received: from mail.kernel.org (pdx-korg-mail-1.web.codeaurora.org
 [172.30.200.123])
	by pdx-korg-patchwork-2.web.codeaurora.org (Postfix) with ESMTP id 03F29913
	for <patchwork-kvm@patchwork.kernel.org>;
 Tue, 12 May 2020 10:33:06 +0000 (UTC)
Received: from vger.kernel.org (vger.kernel.org [23.128.96.18])
	by mail.kernel.org (Postfix) with ESMTP id E121F20731
	for <patchwork-kvm@patchwork.kernel.org>;
 Tue, 12 May 2020 10:33:05 +0000 (UTC)
Authentication-Results: mail.kernel.org;
	dkim=pass (1024-bit key) header.d=redhat.com header.i=@redhat.com
 header.b="MsNC4uVk"
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand
        id S1728085AbgELKdF (ORCPT
        <rfc822;patchwork-kvm@patchwork.kernel.org>);
        Tue, 12 May 2020 06:33:05 -0400
Received: from us-smtp-2.mimecast.com ([207.211.31.81]:59770 "EHLO
        us-smtp-delivery-1.mimecast.com" rhost-flags-OK-OK-OK-FAIL)
        by vger.kernel.org with ESMTP id S1729365AbgELKdE (ORCPT
        <rfc822;kvm@vger.kernel.org>); Tue, 12 May 2020 06:33:04 -0400
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=redhat.com;
        s=mimecast20190719; t=1589279583;
        h=from:from:reply-to:subject:subject:date:date:message-id:message-id:
         to:to:cc:cc:mime-version:mime-version:
  content-type:content-type:content-type:
         content-transfer-encoding:content-transfer-encoding:
         in-reply-to:in-reply-to:references:references;
        bh=FN/9Q7LqiT7yOKM3lyj2/vvjplGjuC96R8PuYHJ2hZI=;
        b=MsNC4uVksMZaeiI/oTccX+OOK0OGB1xJ2ZTo5Sk7VQtHP8nkM1kU+drA5/rjt/dHzgqvMD
        lEtxcVbIRZ1dyyqU0D5dF3Xt+JOzFMkbpHCeH3HVPoJXrwxa3XEhdmEeNVkzj8e8Inkgur
        g+Z9/R+6695OYcGrIj0Mp6AWDz/GJJ0=
Received: from mail-wr1-f71.google.com (mail-wr1-f71.google.com
 [209.85.221.71]) (Using TLS) by relay.mimecast.com with ESMTP id
 us-mta-473-4x6skZNAMFWiIs_LGqWm5A-1; Tue, 12 May 2020 06:33:00 -0400
X-MC-Unique: 4x6skZNAMFWiIs_LGqWm5A-1
Received: by mail-wr1-f71.google.com with SMTP id j16so6709099wrw.20
        for <kvm@vger.kernel.org>; Tue, 12 May 2020 03:33:00 -0700 (PDT)
X-Google-DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
        d=1e100.net; s=20161025;
        h=x-gm-message-state:from:to:cc:subject:date:message-id:in-reply-to
         :references:mime-version:content-transfer-encoding;
        bh=FN/9Q7LqiT7yOKM3lyj2/vvjplGjuC96R8PuYHJ2hZI=;
        b=sRdZitX5IYQpoHULqXkakqqmO82cGxhnGwjAxFIRTPhOBNeK4OSnoBxUGms8kh5nQF
         /EXjgrvSdjupVxYApJ4nj5REfwlI5Te8qVwLMZbBLVFn+RNaOYH9iQ9iP7QXqR5WZRXp
         h2liD0IliwFzxc/13ADiHQUnoEFn0ABzP+9VUdORICaZGlXpW/va+xr0/sJ6Kc1e2WNa
         EpsOr05rwBdrAZHJJJtP+AOnu8CYD9FnUC5C4ug5a2PKjBFKaAGnIxw2Dg+5q8WY9Y0S
         Va+ca1Qsoq2847nV4Ek7nQ0xYNcYncrTqUzAJjJ6xK4u4I/95kU6tu8h6hS0ahhbieiT
         53Ew==
X-Gm-Message-State: AGi0Pubk58Xh54s/byzFGfZfqqh3lEjUvdufkSk6Ucsj2j1ye0Wp8PGH
        ECUFbj0IdAKvAwL7WMnxuOTGbIG5GXDcrmG9mmjcUMrzc5b+LdWrSQJBBnDdpLDfQ1B3ANBJn4T
        eEOxURMH0YIEX
X-Received: by 2002:a1c:a3c1:: with SMTP id
 m184mr18797671wme.91.1589279579783;
        Tue, 12 May 2020 03:32:59 -0700 (PDT)
X-Google-Smtp-Source: 
 APiQypLoVobmiuPPC5yzEHiFpfHm0s+HDrmev5jafrGVEC8Jo2mwsLRcy5dqC4UGFChZXKNb5cxJ4A==
X-Received: by 2002:a1c:a3c1:: with SMTP id
 m184mr18797645wme.91.1589279579565;
        Tue, 12 May 2020 03:32:59 -0700 (PDT)
Received: from x1w.redhat.com (17.red-88-21-202.staticip.rima-tde.net.
 [88.21.202.17])
        by smtp.gmail.com with ESMTPSA id
 v131sm3712043wmb.27.2020.05.12.03.32.58
        (version=TLS1_3 cipher=TLS_AES_256_GCM_SHA384 bits=256/256);
        Tue, 12 May 2020 03:32:59 -0700 (PDT)
From: =?utf-8?q?Philippe_Mathieu-Daud=C3=A9?= <philmd@redhat.com>
To: qemu-devel@nongnu.org
Cc: Markus Armbruster <armbru@redhat.com>, John Snow <jsnow@redhat.com>,
 Paolo Bonzini <pbonzini@redhat.com>, qemu-block@nongnu.org,
 qemu-trivial@nongnu.org,
 =?utf-8?q?Philippe_Mathieu-Daud=C3=A9?= <philmd@redhat.com>,
 Cleber Rosa <crosa@redhat.com>, kvm@vger.kernel.org,
 Eduardo Habkost <ehabkost@redhat.com>,
 =?utf-8?q?Alex_Benn=C3=A9e?= <alex.bennee@linaro.org>,
 Marcelo Tosatti <mtosatti@redhat.com>, Stefan Hajnoczi <stefanha@redhat.com>,
 Fam Zheng <fam@euphon.net>
Subject: [PATCH v4 4/6] scripts/kvm/vmxcap: Use Python 3 interpreter and add
 pseudo-main()
Date: Tue, 12 May 2020 12:32:36 +0200
Message-Id: <20200512103238.7078-5-philmd@redhat.com>
X-Mailer: git-send-email 2.21.3
In-Reply-To: <20200512103238.7078-1-philmd@redhat.com>
References: <20200512103238.7078-1-philmd@redhat.com>
MIME-Version: 1.0
Sender: kvm-owner@vger.kernel.org
Precedence: bulk
List-ID: <kvm.vger.kernel.org>
X-Mailing-List: kvm@vger.kernel.org

Signed-off-by: Philippe Mathieu-Daudé <philmd@redhat.com>
Acked-by: Paolo Bonzini <pbonzini@redhat.com>
---
 scripts/kvm/vmxcap | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/scripts/kvm/vmxcap b/scripts/kvm/vmxcap
index 971ed0e721..6fe66d5f57 100755
--- a/scripts/kvm/vmxcap
+++ b/scripts/kvm/vmxcap
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python3
 #
 # tool for querying VMX capabilities
 #
@@ -275,5 +275,6 @@ controls = [
         ),
     ]
 
-for c in controls:
-    c.show()
+if __name__ == '__main__':
+    for c in controls:
+        c.show()

From patchwork Tue May 12 10:32:37 2020
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 8bit
X-Patchwork-Submitter: =?utf-8?q?Philippe_Mathieu-Daud=C3=A9?=
 <philmd@redhat.com>
X-Patchwork-Id: 11542885
Return-Path: <SRS0=B8Dm=62=vger.kernel.org=kvm-owner@kernel.org>
Received: from mail.kernel.org (pdx-korg-mail-1.web.codeaurora.org
 [172.30.200.123])
	by pdx-korg-patchwork-2.web.codeaurora.org (Postfix) with ESMTP id 22FDF913
	for <patchwork-kvm@patchwork.kernel.org>;
 Tue, 12 May 2020 10:33:12 +0000 (UTC)
Received: from vger.kernel.org (vger.kernel.org [23.128.96.18])
	by mail.kernel.org (Postfix) with ESMTP id 0A96A20733
	for <patchwork-kvm@patchwork.kernel.org>;
 Tue, 12 May 2020 10:33:12 +0000 (UTC)
Authentication-Results: mail.kernel.org;
	dkim=pass (1024-bit key) header.d=redhat.com header.i=@redhat.com
 header.b="FrxQAN8f"
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand
        id S1729506AbgELKdL (ORCPT
        <rfc822;patchwork-kvm@patchwork.kernel.org>);
        Tue, 12 May 2020 06:33:11 -0400
Received: from us-smtp-delivery-1.mimecast.com ([207.211.31.120]:29413 "EHLO
        us-smtp-1.mimecast.com" rhost-flags-OK-OK-OK-FAIL) by vger.kernel.org
        with ESMTP id S1729365AbgELKdK (ORCPT <rfc822;kvm@vger.kernel.org>);
        Tue, 12 May 2020 06:33:10 -0400
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=redhat.com;
        s=mimecast20190719; t=1589279589;
        h=from:from:reply-to:subject:subject:date:date:message-id:message-id:
         to:to:cc:cc:mime-version:mime-version:
  content-type:content-type:content-type:
         content-transfer-encoding:content-transfer-encoding:
         in-reply-to:in-reply-to:references:references;
        bh=ds4l+7fZ3gMORFydzgGtrLwAaPpZWtJvEPP6Ecz3QgY=;
        b=FrxQAN8f4Cm814+g7613LhdhBHZTNGuyM/EKPEyI0l8lmWWzVywpmDTLzQZpjYN77E6Dwp
        +Me0G8QXRLUwhR8ttHM/pO4zTUl20nf9qhKp/uW4y1SDPxjjiCnIhgh3GCe5e/vGR9D/ru
        mAZFRk6qyxRkdNhyz1R8OcptC32kL5Y=
Received: from mail-wm1-f69.google.com (mail-wm1-f69.google.com
 [209.85.128.69]) (Using TLS) by relay.mimecast.com with ESMTP id
 us-mta-151-zgC5UCrsM9-wuqEdoliQjw-1; Tue, 12 May 2020 06:33:05 -0400
X-MC-Unique: zgC5UCrsM9-wuqEdoliQjw-1
Received: by mail-wm1-f69.google.com with SMTP id q5so9752369wmc.9
        for <kvm@vger.kernel.org>; Tue, 12 May 2020 03:33:05 -0700 (PDT)
X-Google-DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
        d=1e100.net; s=20161025;
        h=x-gm-message-state:from:to:cc:subject:date:message-id:in-reply-to
         :references:mime-version:content-transfer-encoding;
        bh=ds4l+7fZ3gMORFydzgGtrLwAaPpZWtJvEPP6Ecz3QgY=;
        b=FneAUTiiHFUNkdzHBq3q9P9D6Wo11GrzAJemRpP6pctXeyjOMorPs0TaTzGSCEL3Xe
         Z5pgjL7MXFPAqzOJY00ptuqkMc4CyMQ47Nyaq0c/QWYOG40ebxxrKVQO8IwmaynKwoLD
         YbcmC2lxlOFOph6AB+EyyRi9LJoq2xpDgKYbER8nNaKkDUaqBCDLs0vDADvq1a7X3WO/
         DyIPfdmMPqMPu1AsSoFDKmMLXuCzl7v9A/YJz9ro1k3LbfKuvOpKh9btT6odA4XpWcx3
         paUvr30il2PZm6deK77jC7pE5Lq2xoQVErxHUk8VTDM4mUhhzfOConnzJim0B5VTFYYe
         C33A==
X-Gm-Message-State: AGi0PuYLae1jg9+MyG4WhP1gjJOtSNqIFYuBaw7xsMF6S6CisXX5C95w
        OvcVnXONy+pnw0Evy1g0oD8GpZiXzFMI4i/7XJU+r5PckPVk3NuUI2I3NefWgLgYCU7hVumsP7v
        4j2RTyo+bp6ok
X-Received: by 2002:a1c:e4c1:: with SMTP id b184mr20242340wmh.4.1589279584721;
        Tue, 12 May 2020 03:33:04 -0700 (PDT)
X-Google-Smtp-Source: 
 APiQypI/laT2LsXrPFPVBKvEk7pWPnx7ibaLAWboB7OwBuZFbrY2ibbD5F30XX0MAqw3dY3xhmeCtA==
X-Received: by 2002:a1c:e4c1:: with SMTP id b184mr20242317wmh.4.1589279584461;
        Tue, 12 May 2020 03:33:04 -0700 (PDT)
Received: from x1w.redhat.com (17.red-88-21-202.staticip.rima-tde.net.
 [88.21.202.17])
        by smtp.gmail.com with ESMTPSA id
 w15sm21193073wrl.73.2020.05.12.03.33.03
        (version=TLS1_3 cipher=TLS_AES_256_GCM_SHA384 bits=256/256);
        Tue, 12 May 2020 03:33:03 -0700 (PDT)
From: =?utf-8?q?Philippe_Mathieu-Daud=C3=A9?= <philmd@redhat.com>
To: qemu-devel@nongnu.org
Cc: Markus Armbruster <armbru@redhat.com>, John Snow <jsnow@redhat.com>,
 Paolo Bonzini <pbonzini@redhat.com>, qemu-block@nongnu.org,
 qemu-trivial@nongnu.org,
 =?utf-8?q?Philippe_Mathieu-Daud=C3=A9?= <philmd@redhat.com>,
 Cleber Rosa <crosa@redhat.com>, kvm@vger.kernel.org,
 Eduardo Habkost <ehabkost@redhat.com>,
 =?utf-8?q?Alex_Benn=C3=A9e?= <alex.bennee@linaro.org>,
 Marcelo Tosatti <mtosatti@redhat.com>, Stefan Hajnoczi <stefanha@redhat.com>,
 Fam Zheng <fam@euphon.net>
Subject: [PATCH v4 5/6] scripts/modules/module_block: Use Python 3 interpreter
 & add pseudo-main
Date: Tue, 12 May 2020 12:32:37 +0200
Message-Id: <20200512103238.7078-6-philmd@redhat.com>
X-Mailer: git-send-email 2.21.3
In-Reply-To: <20200512103238.7078-1-philmd@redhat.com>
References: <20200512103238.7078-1-philmd@redhat.com>
MIME-Version: 1.0
Sender: kvm-owner@vger.kernel.org
Precedence: bulk
List-ID: <kvm.vger.kernel.org>
X-Mailing-List: kvm@vger.kernel.org

Signed-off-by: Philippe Mathieu-Daudé <philmd@redhat.com>
---
 scripts/modules/module_block.py | 31 ++++++++++++++++---------------
 1 file changed, 16 insertions(+), 15 deletions(-)

diff --git a/scripts/modules/module_block.py b/scripts/modules/module_block.py
index f23191fac1..2e7021b952 100644
--- a/scripts/modules/module_block.py
+++ b/scripts/modules/module_block.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python3
 #
 # Module information generator
 #
@@ -10,7 +10,6 @@
 # This work is licensed under the terms of the GNU GPL, version 2.
 # See the COPYING file in the top-level directory.
 
-import sys
 import os
 
 def get_string_struct(line):
@@ -80,19 +79,21 @@ def print_bottom(fheader):
 #endif
 ''')
 
-# First argument: output file
-# All other arguments: modules source files (.c)
-output_file = sys.argv[1]
-with open(output_file, 'w') as fheader:
-    print_top(fheader)
+if __name__ == '__main__':
+    import sys
+    # First argument: output file
+    # All other arguments: modules source files (.c)
+    output_file = sys.argv[1]
+    with open(output_file, 'w') as fheader:
+        print_top(fheader)
 
-    for filename in sys.argv[2:]:
-        if os.path.isfile(filename):
-            process_file(fheader, filename)
-        else:
-            print("File " + filename + " does not exist.", file=sys.stderr)
-            sys.exit(1)
+        for filename in sys.argv[2:]:
+            if os.path.isfile(filename):
+                process_file(fheader, filename)
+            else:
+                print("File " + filename + " does not exist.", file=sys.stderr)
+                sys.exit(1)
 
-    print_bottom(fheader)
+        print_bottom(fheader)
 
-sys.exit(0)
+    sys.exit(0)

From patchwork Tue May 12 10:32:38 2020
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 8bit
X-Patchwork-Submitter: =?utf-8?q?Philippe_Mathieu-Daud=C3=A9?=
 <philmd@redhat.com>
X-Patchwork-Id: 11542887
Return-Path: <SRS0=B8Dm=62=vger.kernel.org=kvm-owner@kernel.org>
Received: from mail.kernel.org (pdx-korg-mail-1.web.codeaurora.org
 [172.30.200.123])
	by pdx-korg-patchwork-2.web.codeaurora.org (Postfix) with ESMTP id 7E34A913
	for <patchwork-kvm@patchwork.kernel.org>;
 Tue, 12 May 2020 10:33:15 +0000 (UTC)
Received: from vger.kernel.org (vger.kernel.org [23.128.96.18])
	by mail.kernel.org (Postfix) with ESMTP id 65A3520722
	for <patchwork-kvm@patchwork.kernel.org>;
 Tue, 12 May 2020 10:33:15 +0000 (UTC)
Authentication-Results: mail.kernel.org;
	dkim=pass (1024-bit key) header.d=redhat.com header.i=@redhat.com
 header.b="OgDDXfc6"
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand
        id S1729541AbgELKdO (ORCPT
        <rfc822;patchwork-kvm@patchwork.kernel.org>);
        Tue, 12 May 2020 06:33:14 -0400
Received: from us-smtp-2.mimecast.com ([207.211.31.81]:24246 "EHLO
        us-smtp-delivery-1.mimecast.com" rhost-flags-OK-OK-OK-FAIL)
        by vger.kernel.org with ESMTP id S1729515AbgELKdN (ORCPT
        <rfc822;kvm@vger.kernel.org>); Tue, 12 May 2020 06:33:13 -0400
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=redhat.com;
        s=mimecast20190719; t=1589279592;
        h=from:from:reply-to:subject:subject:date:date:message-id:message-id:
         to:to:cc:cc:mime-version:mime-version:
  content-type:content-type:content-type:
         content-transfer-encoding:content-transfer-encoding:
         in-reply-to:in-reply-to:references:references;
        bh=eW6bfQ4PBPfgnbJADVXdHyk1e9elM2KC/4eDs8BhUVc=;
        b=OgDDXfc6+l5VAultsME/p+0QOaz/J+R5Ih/DTYhb5YkTxKYnlYYoeaYs89Y8XMjOYG6+fW
        nd2RZ1xJA1ak1PlVDIX1IBfj4wsX8QglMt/TYsHpA11IRhioYP0CfB4uGuq8+vz0RM5URp
        xz99pbI0PV0lteu7a5AlGnAcZnx7X/Y=
Received: from mail-wr1-f70.google.com (mail-wr1-f70.google.com
 [209.85.221.70]) (Using TLS) by relay.mimecast.com with ESMTP id
 us-mta-461-QhwaqcciM02jcHuPyHjZXQ-1; Tue, 12 May 2020 06:33:10 -0400
X-MC-Unique: QhwaqcciM02jcHuPyHjZXQ-1
Received: by mail-wr1-f70.google.com with SMTP id 30so6720181wrq.15
        for <kvm@vger.kernel.org>; Tue, 12 May 2020 03:33:10 -0700 (PDT)
X-Google-DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
        d=1e100.net; s=20161025;
        h=x-gm-message-state:from:to:cc:subject:date:message-id:in-reply-to
         :references:mime-version:content-transfer-encoding;
        bh=eW6bfQ4PBPfgnbJADVXdHyk1e9elM2KC/4eDs8BhUVc=;
        b=HcwVIngizecdc2RsSVktMG+Dsibr/l+ZQz7FCjkpv260aGmO+/syVFlBg5VQBp0JIk
         9xV+tlaLEwb/YRqe/riujMFKk55oQ6tDcvfz/VeeYVhm81qXOSyXevydYS8qsMZbRkND
         lsqnfTbRsOI5fcQ+KyT1d6lxekxZnwaObwJBEjuWCgZMZiIF+92i32C3S+vcjDyaE2iQ
         ouD+EQ0pN+UqC+adiKxn/KZMt0j8cPj3eFjImQ9wLnCmEQurYyyTRDHf6q0EWM6JiJbq
         p+XqPZPSEifiWydv8dTp0FoUtfisccUdKicF/uq9D1Hq3bObmATuajlZcuO0kYSE7B+Y
         2s2Q==
X-Gm-Message-State: AGi0PubCml1jpYfKMheBGfqNKTCGJXQDlxxz4StIE2ZCvX73Ysmf2rXg
        ys2bR87w35+ue/Ry+3HqJUL91y/xi1BWwMuzjM+T8v/MTUAOqjZxCqbhsxW3D2F2e7rENv3fm5T
        DsrSNNuKpUqzX
X-Received: by 2002:a5d:526f:: with SMTP id
 l15mr23961461wrc.367.1589279589596;
        Tue, 12 May 2020 03:33:09 -0700 (PDT)
X-Google-Smtp-Source: 
 APiQypKAy4y1dqGWaojxWwDczmxiyvjplmoiyIK4YplpgxZqaLOv7U32GRAJXJeKUMX6KRAwfFepYQ==
X-Received: by 2002:a5d:526f:: with SMTP id
 l15mr23961434wrc.367.1589279589392;
        Tue, 12 May 2020 03:33:09 -0700 (PDT)
Received: from x1w.redhat.com (17.red-88-21-202.staticip.rima-tde.net.
 [88.21.202.17])
        by smtp.gmail.com with ESMTPSA id
 y3sm21421190wrt.87.2020.05.12.03.33.08
        (version=TLS1_3 cipher=TLS_AES_256_GCM_SHA384 bits=256/256);
        Tue, 12 May 2020 03:33:08 -0700 (PDT)
From: =?utf-8?q?Philippe_Mathieu-Daud=C3=A9?= <philmd@redhat.com>
To: qemu-devel@nongnu.org
Cc: Markus Armbruster <armbru@redhat.com>, John Snow <jsnow@redhat.com>,
 Paolo Bonzini <pbonzini@redhat.com>, qemu-block@nongnu.org,
 qemu-trivial@nongnu.org,
 =?utf-8?q?Philippe_Mathieu-Daud=C3=A9?= <philmd@redhat.com>,
 Cleber Rosa <crosa@redhat.com>, kvm@vger.kernel.org,
 Eduardo Habkost <ehabkost@redhat.com>,
 =?utf-8?q?Alex_Benn=C3=A9e?= <alex.bennee@linaro.org>,
 Marcelo Tosatti <mtosatti@redhat.com>, Stefan Hajnoczi <stefanha@redhat.com>,
 Fam Zheng <fam@euphon.net>
Subject: [PATCH v4 6/6] tests/migration/guestperf: Use Python 3 interpreter
Date: Tue, 12 May 2020 12:32:38 +0200
Message-Id: <20200512103238.7078-7-philmd@redhat.com>
X-Mailer: git-send-email 2.21.3
In-Reply-To: <20200512103238.7078-1-philmd@redhat.com>
References: <20200512103238.7078-1-philmd@redhat.com>
MIME-Version: 1.0
Sender: kvm-owner@vger.kernel.org
Precedence: bulk
List-ID: <kvm.vger.kernel.org>
X-Mailing-List: kvm@vger.kernel.org

Signed-off-by: Philippe Mathieu-Daudé <philmd@redhat.com>
---
 tests/migration/guestperf-batch.py | 2 +-
 tests/migration/guestperf-plot.py  | 2 +-
 tests/migration/guestperf.py       | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/tests/migration/guestperf-batch.py b/tests/migration/guestperf-batch.py
index cb150ce804..f1e900908d 100755
--- a/tests/migration/guestperf-batch.py
+++ b/tests/migration/guestperf-batch.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python3
 #
 # Migration test batch comparison invokation
 #
diff --git a/tests/migration/guestperf-plot.py b/tests/migration/guestperf-plot.py
index d70bb7a557..907151011a 100755
--- a/tests/migration/guestperf-plot.py
+++ b/tests/migration/guestperf-plot.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python3
 #
 # Migration test graph plotting command
 #
diff --git a/tests/migration/guestperf.py b/tests/migration/guestperf.py
index 99b027e8ba..ba1c4bc4ca 100755
--- a/tests/migration/guestperf.py
+++ b/tests/migration/guestperf.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python3
 #
 # Migration test direct invokation command
 #
